### PR TITLE
Correct spelling in swap space release note

### PR DESCRIPTION
**Pull request by MoteHue**

### CODE REVIEW

**Review by seunghun1ee — DISMISSED** [review]
Oops. Thank you!

File: releasenotes/notes/add-alerts-for-swap-availability-75e28ed7f913d1ec.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 features:
   - |
     Added two alerts (Warning and critical) that are triggered when the ratio
-    of (free_swap_sppace / total_swap_space) is below thresholds.
+    of (free_swap_space / total_swap_space) is below thresholds.
     Each threshold can be modified by alterting value of
```
Comment (priteau) [inline releasenotes/notes/add-alerts-for-swap-availability-75e28ed7f913d1ec.yaml]: You should fix the typo of altering while here.

**Review by priteau — CHANGES_REQUESTED** [review]

File: releasenotes/notes/add-alerts-for-swap-availability-75e28ed7f913d1ec.yaml (hunk 1 above)
Comment (MoteHue) [reply releasenotes/notes/add-alerts-for-swap-availability-75e28ed7f913d1ec.yaml]: Done

File: releasenotes/notes/add-alerts-for-swap-availability-75e28ed7f913d1ec.yaml [hunk 2]
```diff
@@ -2,8 +2,8 @@
 features:
   - |
     Added two alerts (Warning and critical) that are triggered when the ratio
```
Comment (Alex-Welsh) [inline releasenotes/notes/add-alerts-for-swap-availability-75e28ed7f913d1ec.yaml]: nit: warning is capitalised but critical is not?
Not worth holding up this PR for but thought I'd mention it while we're here

**Review by Alex-Welsh — DISMISSED** [review]

File: releasenotes/notes/add-alerts-for-swap-availability-75e28ed7f913d1ec.yaml (hunk 2 above)
Comment (MoteHue) [reply releasenotes/notes/add-alerts-for-swap-availability-75e28ed7f913d1ec.yaml]: Fixed

**Review by Alex-Welsh — APPROVED** [review]

**Pull request merged**